core/arch/arm/kernel/generic_boot.c

```diff
@@ -1179,7 +1178,6 @@ void __weak paged_init_primary(unsigned long fdt)
 #ifndef CFG_VIRTUALIZATION
 	init_tee_runtime();
 #endif
-	release_external_dt();
```

Comment:
This changes the order, prior to this patch `release_external_dt()` was called before `core_mmu_init_virtualization()`, now it called after (as part of `call_finalcalls()`). Is it OK?

Reply:
As far as `release_external_dt()` shall be called before non-secure world in entered, it looks good to me.

Reply:
But @lorc may have some comment about it...

Reply:
It doesn't seem they have much to do with each other. @lorc?